launcher/ui/dialogs/ExportToModListDialog.ui

```diff
@@ -7,7 +7,7 @@
     <x>0</x>
     <y>0</y>
     <width>650</width>
-    <height>446</height>
+    <height>522</height>
```

Comment:
Did this need change?

Reply:
I used QtDesigner and did the change automatically.
I can remove it if you want.

Reply:
You didn't accidentally resize the window then? it just changed automatically?

Reply:
yes, I only played with the Size constrains of a few elements